### PR TITLE
Create Pro Santé Connect

### PR DESCRIPTION
**Pull request by etienne0101**

### CODE REVIEW

File: _data/api/pro-sante-connect.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: API Pro Santé Connect
+tagline: Permettre l'authentification des professionnels de la santé
```
Comment (XavierJp) [inline _data/api/pro-sante-connect.md:3]: I prefer to use a verb that describe the most frequent use of with this API. Eg. Identifier, Authentifier or Accéder aux données

We can add a mention of RPPS or the id used to identified a "professionnel"

File: _data/api/pro-sante-connect.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+external_site: https://esante.gouv.fr/
+is_open: -1 # -1 means API not open
+datapass_link: https://esante.gouv.fr/
```
Comment (XavierJp) [inline _data/api/pro-sante-connect.md:6]: We can leave it empty for now, or comment. To comment, add a # at the beginning of the line

File: _data/api/pro-sante-connect.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      Seules les administrations et les éditeurs de logiciels sont habilitées à utiliser l'API Pro Santé Connect ou intégrer le bouton Pro Santé Connect.
+
+      Si vous cherchez à contacter l'équipe Pro Santé Connect, vous pouvez [contacter l'Agence Numérique en Santé](https://esante.gouv.fr/assistance)
+
+      <Button href="/rechercher-api">Ou vous pouvez rechercher une autre API</Button>
+  - who:
+      - Une entreprise
+    is_eligible: 0
+    description: |
+      Seuls les éditeurs de logiciels proposant une solution web adressée aux professionnels de santé peuvent demander à se raccorder à PRO Santé Connect.
```
Comment (XavierJp) [inline _data/api/pro-sante-connect.md:21]: We miss the condition "french"

Let's choose one spelling for Pro Santé Connect . According to their website I would stick to Pro Santé Connect and avoid PRO

File: _data/api/pro-sante-connect.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      <NextSteps />
+      <Button href="https://franceconnect.gouv.fr/partenaires">Remplir une demande</Button>
```
Comment (XavierJp) [inline _data/api/pro-sante-connect.md:23]: We can use `href="mailto:prosanteconnect.editeurs@esante.gouv.fr"`

File: _data/api/pro-sante-connect.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  - who:
+      - Une collectivité ou une administration
+    is_eligible: 1
+    description: |
+      Vous pouvez réaliser une demande d'accès à l'API. 
+
```
Comment (XavierJp) [inline _data/api/pro-sante-connect.md:29]: I advise to always have a CTA here

`<NextSteps />
<Button href="mailto:prosanteconnect.editeurs@esante.gouv.fr">Remplir une demande</Button>`

File: _data/api/pro-sante-connect.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  - who:
+      - Une collectivité ou une administration
+    is_eligible: 1
+    description: |
+      Vous pouvez réaliser une demande d'accès à l'API. 
+      
+  - Un éditeur de logiciels
+    is_eligible: 0
+    description: |
+      Tout éditeur proposant une solution web adressée aux professionnels de santé et qui souhaite pouvoir authentifier des professionnels de santé peut demander à se raccorder à PRO Santé Connect.
```
Comment (XavierJp) [inline _data/api/pro-sante-connect.md:33]: Add french, fix Pro/PRO

File: _data/api/pro-sante-connect.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+partners:
+  - Ministère des Solidarités et de la Santé (MSS)
+producer: ans
+keywords:
+  - Authentification
+  - Identification
+  - SSO
+  - Santé
+  - Professionnels
+  - Editeurs
```
Comment (XavierJp) [inline _data/api/pro-sante-connect.md:46]: Perfect. I would maybe add `Connect` and `RPPS`, `adeli`, ìdNat`

File: _data/api/pro-sante-connect.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+themes:
+  - Santé
+contact_link: prosanteconnect.editeurs@esante.gouv.fr
+doc_tech_external: https://integrateurs-cps.asipsante.fr/pages/prosanteconnect/documentation-fs
+last_update: 22/03/2021
+---
+
+L‘API Pro Santé connect permet à des administrations, des éditeurs de logiciels ou des entreprises d’ajouter un bouton Pro Santé Connect, pour recueillir des données d’identité fiables et ainsi identifier les utilisateurs de leur service en ligne (en vue d’une entrée en relation ou connexion).
+
+### A quoi ca sert ?
```
Comment (XavierJp) [inline _data/api/pro-sante-connect.md:56]: Prefer 
A quoi sert cette API ?

File: _data/api/pro-sante-connect.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+
+PRO Santé Connect est un fournisseur d’identité (FI) au standard OpenID. Il accepte comme « credential » d’authentification la carte CPS et une e-carte CPS qui est une application téléphone. L’assertion d’authentification OpenIDConnect « remonte » au fournisseur de service (FS) demandeur de l’authentification les traits d’identités classiques plus l’identité sectorielle (profession, savoir-faire, situation d’exercice, …) afin que celui-ci puisse, au-delà de la simple authentification, gérer tout ou partie de son contrôle d’accès . Ces éléments sont fournis par [notre annuaire](https://annuaire.sante.fr) et incorporé dans le jeton OpenIdConnect.
+                                                                                                                          |
+
+### Comment ça marche ?
```
Comment (XavierJp) [inline _data/api/pro-sante-connect.md:61]: Prefer 
Données accessibles dans l'API

File: _data/api/pro-sante-connect.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+
+**L'identité pivot**.
+
+| Type       | Données                      |
+| --------- | -------------------------------- |
+| Données d'identification       | nom, prénom, idNat (Adeli ou RPPS) |
```
Comment (XavierJp) [inline _data/api/pro-sante-connect.md:67]: I would split it this way
| Identifiant unique (idNat) | Adeli ou RPPS |
| état civil | nom et prénom |

**Review by XavierJp — CHANGES_REQUESTED** [review]
Great first Pull Request, a few details here and there but bravo !

**Comment by etienne0101** [comment]
Thanks @XavierJp for commenting, It should be ok now

**Comment by XavierJp** [comment]
@etienne0101 je ferme cette vieille PR qui est redondante avec #696

**Pull request closed without merging**